care/hcx/api/viewsets/gateway.py

```diff
@@ -138,42 +139,21 @@ def make_claim(self, request):
                 )
             )
 
-        diagnoses = []
-        if len(consultation.icd11_diagnoses):
-            diagnoses = list(
-                map(
-                    lambda diagnosis: {
-                        "id": str(uuid()),
-                        "label": diagnosis.label.split(" ", 1)[1],
-                        "code": diagnosis.label.split(" ", 1)[0] or "00",
-                        "type": "clinical",
-                    },
-                    list(
-                        map(
-                            lambda icd_id: ICDDiseases.by.id[icd_id],
-                            consultation.icd11_diagnoses,
-                        )
-                    ),
-                )
-            )
-
-        if len(consultation.icd11_provisional_diagnoses):
-            diagnoses = list(
-                map(
-                    lambda diagnosis: {
-                        "id": str(uuid()),
-                        "label": diagnosis.label.split(" ", 1)[1],
-                        "code": diagnosis.label.split(" ", 1)[0] or "00",
-                        "type": "admitting",
-                    },
-                    list(
-                        map(
-                            lambda icd_id: ICDDiseases.by.id[icd_id],
-                            consultation.icd11_provisional_diagnoses,
-                        )
-                    ),
-                )
+        diagnoses = list(
+            map(
+                lambda diagnosis: {
+                    "id": str(uuid()),
+                    "label": diagnosis.label.split(" ", 1)[1],
+                    "code": diagnosis.label.split(" ", 1)[0] or "00",
+                    "type": "clinical",
```

Comment:
Are we marking all as clinical?

Reply:
@khavinshankar

Reply:
<img width="1173" alt="image" src="https://github.com/coronasafe/care/assets/25143503/a65dfe75-55d1-4fe7-8105-ec0a4957dd12">

Reply:
for a principal diagnosis, we can go `principal`, and differential as `differential` and rest as `clinical`